bitnami/mariadb/README.md

```diff
@@ -553,4 +553,4 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
\ No newline at end of file
+limitations under the License.
```

Comment:
Hi @ebCrypto,

Could you revert this line break change?

Reply:
If you `Display the rich diff`
there is only one code change related to the database base:

![image](https://user-images.githubusercontent.com/44279886/203677757-4b3e57d0-d058-4584-9404-98804e56ea47.png)

Reply:
Seems the rich this omits this kind of changes. However, I'm afraid your IDE is still adding a new line at the end of the file. As this as it is an unrelated change and also for consistency purposes we would need to remove it.

```
lorenzoa@lorenzoa-a01 charts-ebCrypto % git diff HEAD^
diff --git a/bitnami/mariadb/README.md b/bitnami/mariadb/README.md
index c719765af..cfc7e8bd3 100644
--- a/bitnami/mariadb/README.md
+++ b/bitnami/mariadb/README.md
.
.
.
.
-limitations under the License.
\ No newline at end of file
+limitations under the License.

```

Reply:
Ha I understand what you are saying now. It makes sense. Yet, I made the update directly on GitHub.com so it seems like bitnami can't support that. At this point it probably makes sense for you to close this PR and correct the documentation with a compatible IDE.

Reply:
May I could interact in your fork and branch to correct it? You will still have the ownership of the changes in this way.